pages/api/scholarship-data.js

```diff
@@ -32,8 +39,13 @@ export default async function handler(req, res) {
         scholarship.Category.length === 0,
       (scholarship) =>
         !familyIncome ||
-        scholarship["Family Income"] === null ||
-        scholarship["Family Income"] <= familyIncome,
+        scholarship["Family Income (in LPA)"] === null ||
+        scholarship["Family Income (in LPA)"] >= familyIncome,
```

Comment:
why change the logic here

Reply:
It was wrong previously, 

example: 
 Scholarship requires 4 lakhs income and user entered 10 lakh then not shown